### ISSUE TITLE
Fetch dependencies without compiling them

### ISSUE THREAD

**Issue by fenollp** [ninenines/erlang.mk#560]
But in such a way that they can be compiled at a later step.

We need this in order to package an Erlang release into DEBs/RPMs (an `src` folder that incorporates `deps/`).

I couldn't find documentation about this & am wondering how people usually solve this issue.

**Comment by essen** [comment]
Did you try my suggestion on IRC? Perhaps miss it?

Try defining SKIP_DEPS=1 and then add $(ALL_DEPS_DIRS) to your custom command (or not custom, as long as SKIP_DEPS=1 is set you can redefine deps:: to do what you want). Probably need to do it after include erlang.mk.

``` make
SKIP_DEPS=1

include erlang.mk

deps:: $(ALL_DEPS_DIRS)
    your_custom_commands_here
```

**Comment by dumbbell** [comment]
FWIW, this feature is included #362, through the targets `make fetch-deps`, `make fetch-test-deps` and so on.

**Comment by essen** [comment]
The "make fetch-deps" is in. Enjoy!
